src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/presubmit/UpdateHearingRequestHandler.java

```diff
@@ -82,13 +88,14 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
                                                                 .values())
                     .map(hearingChannel -> new Value(hearingChannel.name(), hearingChannel.getLabel()))
                     .toList();
-                asylumCase.write(HEARING_CHANNEL, new DynamicList(
+                asylumCase.write(REQUEST_HEARING_CHANNEL, new DynamicList(
                     new Value(
                         hearingResponse.getHearingDetails().getHearingChannels().get(0),
                         hearingResponse.getHearingDetails().getHearingChannelDescription()
                     ),
                     hearingChannels
                 ));
+                asylumCase.write(HEARING_LOCATION, locationRefDataService.getHearingLocationsDynamicList());
```

Comment:
You are reusing an existing hearing_location field and here you are overriding it. Are you sure about this?